src/planner/vectorfield/MetaSkeletonStateSpaceVectorField.cpp

```diff
@@ -0,0 +1,65 @@
+#include <dart/dynamics/BodyNode.hpp>
+#include <aikido/constraint/JointStateSpaceHelpers.hpp>
+#include <aikido/planner/vectorfield/MetaSkeletonStateSpaceVectorField.hpp>
+
+namespace aikido {
+namespace planner {
+namespace vectorfield {
+
+//==============================================================================
+MetaSkeletonStateSpaceVectorField::MetaSkeletonStateSpaceVectorField(
+    aikido::statespace::dart::MetaSkeletonStateSpacePtr stateSpace)
+  : VectorField(stateSpace)
+  , mMetaSkeletonStateSpace(stateSpace)
+  , mMetaSkeleton(stateSpace->getMetaSkeleton())
+{
+  if (mMetaSkeleton->getPositionUpperLimits()
+      == mMetaSkeleton->getPositionLowerLimits())
+  {
+    throw std::invalid_argument("State space volume zero");
+  }
+  if (mMetaSkeleton->getVelocityUpperLimits()
+      == mMetaSkeleton->getVelocityLowerLimits())
+  {
+    throw std::invalid_argument("Velocity space volume zero");
+  }
```

Comment:
I don't think these checks are necessary. Bounds should be handled entirely by the `Testable` passed by the user.